### ISSUE TITLE
`either-else` causes silent translation failure

### ISSUE THREAD

**Issue by hwayne** [tlaplus/tlaplus#398]
```
(*--algorithm foo
begin
    either 
      skip;
    else
end algorithm; *)
```
This silently fails translation without raising an error, and without changing the output translation.

**Comment by quaeler** [comment]
There is an `Error` being issued out of `PcalDebug`, but this is resulting in no hint to the user. Will address.

**Comment by quaeler** [comment]
Addressed with 47c0439
